src/views/components/selects/SelectText.tsx

```diff
@@ -21,6 +22,7 @@ export const SelectText = ({ fileId, onChange, noLabel, undefValueOption }: Sele
   // eslint-disable-next-line react-hooks/exhaustive-deps
   const optionals = useMemo(() => ({ deletedOption: t('texts_deleted') }), []);
 
+  if (name) return <StudioDropDown name={name} value={fileId} options={options} onChange={onChange} optionals={optionals} />;
```

Comment:
I don't understand why you added a name property. I think it works without this prop.